plugins/azure_blob_storage/icon_azure_blob_storage/actions/delete_blob/action.py

```diff
@@ -0,0 +1,28 @@
+import insightconnect_plugin_runtime
+from .schema import DeleteBlobInput, DeleteBlobOutput, Input, Output, Component
+
+
+# Custom imports below
+
+
+class DeleteBlob(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="delete_blob", description=Component.DESCRIPTION, input=DeleteBlobInput(), output=DeleteBlobOutput()
+        )
+
+    def run(self, params: dict = None):
+        delete_type_permanent = self.connection.api_client.delete_blob(
+            container_name=params.get(Input.CONTAINER_NAME),
+            blob_name=params.get(Input.BLOB_NAME),
+            snapshot_id=params.get(Input.SNAPSHOT_ID, None),
+            version_id=params.get(Input.VERSION_ID, None),
+            delete_snapshots=params.get(Input.SNAPSHOTS, {}),
+            additional_headers=params.get(Input.ADDITIONAL_HEADERS, {}),
+        )
+        delete_type = "soft" if delete_type_permanent == "false" else "permanent"
```

Comment:
I haven't got any "soft" response from the API but I thought it's possible and according to ticket requirements I have implemented it as described in the ticket. 
```
Output:

{"success": true,
 "message": "Delete blob was successful",
 "deletetype":permanent/soft
} 
 Note: Map the delete-type-permanent to x-ms-delete-type-permanent header response.
 ```
 
 But if the documentation says it's not possible in this version of API maybe it would be better to delete this output or just return "permanent".
 @jkaluza-rapid7 @blaxminarayan-r7 Please let me know what do you think about it